programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -1791,6 +1801,26 @@ declare_syscall!(
     }
 );
 
+declare_syscall!(
+    /// Read remaining compute units
+    SyscallRemainingComputeUnits,
+    fn inner_call(
+        invoke_context: &mut InvokeContext,
+        _arg1: u64,
+        _arg2: u64,
+        _arg3: u64,
+        _arg4: u64,
+        _arg5: u64,
+        _memory_mapping: &mut MemoryMapping,
+    ) -> Result<u64, Error> {
+        let budget = invoke_context.get_compute_budget();
+        consume_compute_meter(invoke_context, budget.syscall_base_cost)?;
```

Comment:
we should probably go ahead and give this its own cost field for maintainability's sake

Reply:
I called this `ComputeBudget::get_remaining_compute_units_cost` and made it 100 - sound good?

Reply:
seem's like we didn't deem the base cost worthy of a constant for w/e reason, so `100` works for me